From fc1deaf77afd3fc767e960c6bf2ee7463eeb3321 Mon Sep 17 00:00:00 2001
From: Jonathan Lichtner
 <195780826+JonathanLichtnerAMD@users.noreply.github.com>
Date: Tue, 17 Jun 2025 23:07:21 +0000
Subject: [PATCH 2/2] Skip some unneeded content when building for MIOpen

These aren't needed by MIOpen and building them in TheRock causes
compile issues.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b0fc72523..6e032a30c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -634,7 +634,7 @@ option(BUILD_MHA_LIB "Build the static library for flash attention" OFF)
 
 add_subdirectory(library)
 
-if(NOT GPU_ARCHS AND USER_GPU_TARGETS)
+if(NOT GPU_ARCHS AND USER_GPU_TARGETS AND NOT MIOPEN_REQ_LIBS_ONLY)
    rocm_package_setup_component(tests
         LIBRARY_NAME composablekernel
         PACKAGE_NAME tests # Prevent -static suffix on package name
-- 
2.25.1

